src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -504,35 +504,6 @@ string project
             return await CreateMasterReviewAsync(user, codeFile, originalFileName, label, memoryStream, compareAllRevisions);
         }
 
-        public async Task<List<ServiceGroupModel>> GetReviewsByServicesAsync(ReviewType filterType)
```

Comment:
curious on what this was for,...

Reply:
We once tried to show reviews grouped by service name